nwbwidgets/test/test_processing_module_widgets.py

```diff
@@ -7,52 +7,39 @@
 from pynwb.behavior import Position, SpatialSeries
 from pynwb import ProcessingModule
 from nwbwidgets.view import default_neurodata_vis_spec
+import unittest
 
 
-
-def test_processing_module():
+class ProcessingModuleTestCase(unittest.TestCase):
+    def setUp(self):
+        spatial_series = SpatialSeries(name='position',
+                                   data=np.linspace(0, 1, 20),
+                                   rate=50.,
+                                   reference_frame='starting gate')
+        self.position = Position(spatial_series=spatial_series)
     
-    start_time = datetime(2020, 1, 29, 11, tzinfo=tzlocal())
-    nwbfile = NWBFile(session_description='Test Session',  
-                  identifier='NWBPM',  
-                  session_start_time=start_time)
+    def test_processing_module(self):
 
-    spatial_series = SpatialSeries(name='position',
-                               data=np.linspace(0, 1, 20),
-                               rate=50.,
-                               reference_frame='starting gate')
-    position = Position(spatial_series=spatial_series)
+        start_time = datetime(2020, 1, 29, 11, tzinfo=tzlocal())
+        nwbfile = NWBFile(session_description='Test Session',  
+                      identifier='NWBPM',  
+                      session_start_time=start_time)
 
-    behavior_module = ProcessingModule(name='behavior',
-                                                   description='preprocessed behavioral data')
-    nwbfile.add_processing_module(behavior_module)
+        behavior_module = ProcessingModule(name='behavior',
+                                                       description='preprocessed behavioral data')
+        nwbfile.add_processing_module(behavior_module)
 
-    nwbfile.processing['behavior'].add(position)
+        nwbfile.processing['behavior'].add(self.position)
 
-    processing_module(nwbfile.processing['behavior'], default_neurodata_vis_spec)
+        self.processing_module(nwbfile.processing['behavior'], default_neurodata_vis_spec)
 
+    def test_nwb2widget(self):
 
-def test_nwb2widget():
+        self.nwb2widget(self.position, default_neurodata_vis_spec)

```

Comment:
`nwb2widget` is not a method or attribute of this class. Should not have `self`.